Assets/MRTK/SDK/Features/UX/Scripts/Pointers/LinePointer.cs

```diff
@@ -117,6 +117,11 @@ public override void OnPreSceneQuery()
             {
                 if (!IsInteractionEnabled)
                 {
+                    LineBase.enabled = false;
```

Comment:
I think we rely on OnPostSceneQuery() to handle turning on/off the LineRenderer. Handling it here is a redundant and makes it harder to determine who was responsible for setting the visibility of the LinePointer's visual elements,

Reply:
I went ahead and removed this for now. I'll keep digging in at a later date.